core/tests/bank_forks.rs

```diff
@@ -337,16 +337,17 @@ mod tests {
     #[test]
     fn test_slots_to_snapshot() {
         solana_logger::setup();
-        for add_root_interval in 1..10 {
+        let num_set_roots = MAX_CACHE_ENTRIES * 2;
```

Comment:
Maybe we should replace `MAX_CACHE_ENTRIES`  with some fixed value so this tests' running time doesn't spike if we change this value

Reply:
Is it not important to the test to cross that value's boundary?

Reply:
ah yeah you're right, it's important here to test that the status cache keeps around only the latest MAX_CACHE_ENTRIES number of roots

Reply:
nits: this variable could be a `const`. i.e. `const NUM_SET_ROOTS = ...`.